src/main/java/org/example/check/SampleCheck.java

```diff
@@ -13,10 +13,15 @@
 
 
 /**
- * This class is meant to demonstrate a simple custom check for Teamscale. It
- * creates a new finding for every method encountered in the system.
+ * This class is meant to demonstrate a simple custom check for Teamscale. The
+ * description of a check can be defined in the Check annotation, or a separate
+ * Markdown file can be created with the name of the check in the
+ * 'resources/check-descriptions' folder. This check has its description at
+ * 'resources/check-descriptions/Sample Check 1.md'.
```

Comment:
I would suggest to reformulate this a bit and to point out the special treatment of some characters
```
The description of a check is defined in `src/main/resources/check-descriptions/Sample Check 1.md`. The name of the file has to match the name of the check. <Describe here which special characters are replaced in the file name>. Alternatively the description can also be given as `description` attribute in the `@Check` annotation.
```

Reply:
Good point, thanks! 👍